pan3d/ui.py

```diff
@@ -1,16 +1,9 @@
-from trame.ui.vuetify import SinglePageWithDrawerLayout
 from trame.widgets import html, vuetify
 from pyvista.trame.ui import plotter_ui
 
 
-# Create single page layout type
-# (FullScreenPage, SinglePage, SinglePageWithDrawer)
-def initialize(server):
-    state, ctrl = server.state, server.controller
-    state.trame__title = "Pan3D Viewer"
-
-    with SinglePageWithDrawerLayout(server) as layout:
-        # Toolbar
+def initialize(layout, ctrl):
```

Comment:
That could be part of the main class

Reply:
I agree but I also think the file would be pretty long if we combine them. Is it acceptable to keep them separate and just restructure what we send to the ui creation function?

Reply:
Resolving because the UI components don't use the controller anymore